### DISCUSSION TITLE
order of autoarima params  return from sf.fitted_[0][0].model_   {'arma': (1, 0, 0, 1, 12, 1, 1)}

### DISCUSSION THREAD

**Discussion by hypercuberajesh** [Nixtla/statsforecast#492] [Q&A]
Hello 👋,
can any one tell what is the order of autoarima params  return from fitted_ ??
like  
"sf.fitted_[0][0].model_ returns dictionary with 
 {'arma': (1, 0, 0, 1, 12, 1, 1)} ".

Thank you.

**Comment by WilsonDog** [comment]
Hi @YallaRajesh ,
Do you mean what each element of the tuple stands for? If so, my best guess is p, q, P, Q, s, d, D. This means AR, MA, seasonal AR and seasonal MA coefficients, plus the period and the number of non-seasonal and seasonal differences.
